lua/nvcommunity/tools/telescope-fzf-native/init.lua

```diff
@@ -0,0 +1,16 @@
+---@type NvPluginSpec
+local spec = {
+  {
+    "nvim-telescope/telescope.nvim",
+    dependencies = {
+      {
+        "nvim-telescope/telescope-fzf-native.nvim",
+        build = "make",
+      },
+    },
+    opts = {
+      extensions_list = { "fzf" },
```

Comment:
it should be fzf themes terms, or else default extensions wont load

Reply:
According to the documentation of lazy.nvim, it will merge with the parent spec.`opts, dependencies, cmd, event, ft and keys are always merged with the parent spec.`

Reply:
@kayuxx  hmm before i thought it wont cuz this isnt a key/val table, its a list :thinking: @BrunoKrugel

Reply:
I have no idea, maybe do a deep merge or use 
`opts = function() require("telescope").load_extension "fzf" end`

Reply:
@BrunoKrugel  just realized lazy merges items of both parent & child lists :flushed: 

so i think we're gg

just tested with https://github.com/ghassan0/telescope-glyph.nvim